uqcsbot/models.py

```diff
@@ -9,14 +9,20 @@ class Channel(Base):
     id = Column("id", BigInteger, primary_key=True, nullable=False)
     name = Column("name", String, nullable=False)
     joinable = Column("joinable", Boolean)
+    emoji = Column("emoji", String, nullable=False)
```

Comment:
Hmm, since I manually modified this table with sql would any further migrations be necessary do you think? Things seem to be okay in my test env. Happy to take at look at using Alembic for the schema migration if more work is needed. Thanks.

Reply:
I think for now that should be fine, Alembic and schema migrations are not a small task and I think this is probably a one off event anyway.